securitycenter/snippets_v2/mute_findings_v2_test.py

```diff
@@ -67,14 +67,14 @@ def test_set_unmute_finding(finding):
     assert response.mute.name == "UNMUTED"
 
 
-# @backoff.on_exception(
-#     backoff.expo, (InternalServerError, ServiceUnavailable, NotFound), max_tries=3
-# )
-# def test_bulk_mute_findings(finding):
-#     # Mute findings that belong to this project.
-#     response = mute_findings_v2.bulk_mute_findings(
-#         f"organizations/{ORGANIZATION_ID}",
-#         "global",
-#         f'resource.project_display_name="{ORGANIZATION_ID}"',
-#     )
-#     assert response.done
+@backoff.on_exception(
+    backoff.expo, (InternalServerError, ServiceUnavailable, NotFound), max_tries=3
+)
+def test_bulk_mute_findings(finding):
+    # Mute findings that belong to this project.
+    response = mute_findings_v2.bulk_mute_findings(
+        f"organizations/{ORGANIZATION_ID}",
+        "global",
+        f'resource.project_display_name="{ORGANIZATION_ID}"',
```

Comment:
be consistent with the quotes. Instead of switching the last line (78) as single quotes, use the outer quotes as double and use the inner one as single quotes.

Reply:
Addressed.